README.md

```diff
@@ -35,6 +35,54 @@ Please follow the steps described in the main repository to build the connector:
 ## Getting Started
 
 For getting started please check the [tutorial example](https://github.com/debezium/debezium-examples/tree/master/tutorial#using-cassandra).
+
+## Running the Debezium Cassandra connector standalone or inside Debezium Server
+
+Running with java 11+ requires setting the following java options at startup trough the JAVA_OPTS environment variable or equivalent:
+
+```
+JAVA_OPTS="$JAVA_OPTS --add-exports java.base/jdk.internal.misc=ALL-UNNAMED --add-exports java.base/jdk.internal.ref=ALL-UNNAMED --add-exports java.base/sun.nio.ch=ALL-UNNAMED --add-exports java.management.rmi/com.sun.jmx.remote.internal.rmi=ALL-UNNAMED --add-exports java.rmi/sun.rmi.registry=ALL-UNNAMED --add-exports java.rmi/sun.rmi.server=ALL-UNNAMED --add-exports java.sql/java.sql=ALL-UNNAMED  --add-opens java.base/java.lang.module=ALL-UNNAMED --add-opens java.base/jdk.internal.loader=ALL-UNNAMED --add-opens java.base/jdk.internal.ref=ALL-UNNAMED --add-opens java.base/jdk.internal.reflect=ALL-UNNAMED --add-opens java.base/jdk.internal.math=ALL-UNNAMED --add-opens java.base/jdk.internal.module=ALL-UNNAMED --add-opens java.base/jdk.internal.util.jar=ALL-UNNAMED --add-opens jdk.management/com.sun.management.internal=ALL-UNNAMED --add-opens=java.base/java.io=ALL-UNNAMED --add-opens java.base/sun.nio.ch=ALL-UNNAMED"
+```
+
+## Sample of `application.properties` for running inside Debezium Server
+
+```
+# Sink
+debezium.sink.type=redis
+debezium.sink.redis.address=localhost:6379
+
+# Connector
```

Comment:
I do not think this whole Cassandra section is relevant anymore after we switched to Cassandra Java connector of Datastax of version 4. Whole Cassandra related configuration is in a special file driver reacts on and here it is just a property which references that file.